cmake/scripts/ROOTConfig.cmake.in

```diff
@@ -63,7 +63,7 @@ get_filename_component(_thisdir "${CMAKE_CURRENT_LIST_FILE}" PATH)
 @ROOT_BINARY_DIR_SETUP@
 
 # Python version used to build ROOT
-set(ROOT_PYTHON_VERSION @PYTHON_VERSION_STRING@)
+set(ROOT_PYTHON_VERSION @PYTHON_VERSION_STRING_Development_Main@)
```

Comment:
So now `ROOT_PYTHON_VERSION` will be the version of the highest PyROOT, right? The Python version which is used by ROOT itself for its internal things is not advertised anymore. Fine with me.